test/tracing/frcnn/test_frcnn_tracing.cpp

```diff
@@ -0,0 +1,25 @@
+#include <torch/torch.h>
+#include <torch/script.h>
+
+
+int main() {
+    torch::DeviceType device_type;
+    device_type = torch::kCPU;
+
+    torch::jit::script::Module module;
+    try {
+        // Deserialize the ScriptModule from a file using torch::jit::load().
+        module = torch::jit::load("fasterrcnn_resnet50_fpn.pt");
+    }
+    catch (const c10::Error& e) {
+        std::cerr << "error loading the model\n";
+        return -1;
+    }
+
+    std::vector<torch::jit::IValue> inputs;
+    inputs.push_back(torch::rand({3, 256, 275}));
+    inputs.push_back(torch::rand({3, 256, 275}));
```

Comment:
Can you feed those inputs to the model so that we can make sure that it runs as expected? Or does it take too long to run?